sphinx/tutorial/cylc/furthertopics/queues.rst

```diff
@@ -89,21 +78,20 @@ Replace the ``[scheduling]`` line with:
 
 Make sure you are in the root directory of your suite.
 
-Run the suite using:
-
-.. code-block:: console
+Run the suite using::
 
    rose suite-run
```

Comment:
Rather unnecessary repetition of this command as in lines 51-54. Otherwise this page is very clear and I can't spot any issues or minor errors.